scripts/copy-ra-oss-docs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+echo "Updating the documentation to version $VERSION"
+echo "Copying to the root folder..."
+cp ./docs/*.html ${RA_DOC_PATH}
```

Comment:
additional check to ensure the `RA_DOC_PATH` and `VERSION` env vars are provided could be a welcome addition (to avoid messing up your workspace)

Reply:
Added